Docker installation manual

```diff
@@ -0,0 +1,113 @@
+Docker and Docker Compose installation manual
```

Comment:
this file doesn't make sense in here, it  was contributed into https://github.com/CodeForPoznan/Community/blob/master/Docker%20installation%20manual.md